backend/corpora/lambdas/api/v1/submission.py

```diff
@@ -1,13 +1,28 @@
-def get_submissions_list(query_user_uuid: str):
-    raise NotImplementedError
+from flask import make_response, jsonify
+
+from ....common.utils.exceptions import AuthorizationError
+from ....common.entities import Project
+
+
+def get_submissions_list(query_user_uuid: str = ""):
+    results = Project.list_submissions()
+    for result in results:
+        result["owner_id"] = result.pop("owner")
+    result = dict(submissions=results)
+    return make_response(jsonify(result), 200)
 
 
 def create_new_submission(request_body: dict):
     raise NotImplementedError
 
 
-def get_submission_details(path_project_uuid: str):
-    raise NotImplementedError
+def get_submission_details(project_uuid: str):
+    project = Project.get_submission(project_uuid)
+    if project:
+        result = project.reshape_for_api()
+        return make_response(jsonify(result), 200)
+    else:
+        raise AuthorizationError()
```

Comment:
Add additional details to the message to elucidate the specific place that this failed?

Reply:
This an error for the frontend. I'm only providing the information that is needed to the user. We can log more information on the backend if it's needed.